star_profile.cpp

```diff
@@ -127,20 +151,20 @@ void ProfileWindow::OnPaint(wxPaintEvent& WXUNUSED(evt))
 
     int i;
     int *profptr;
-    wxString label;
+    string profileLabel;

```

Comment:
using std::string will introduce some odd implicit conversions, and I'm not sure it will produce the right results for all locales.  Better to stick with wxString since a) the return type of _() is wxString, and b) the %s in wxString::Format() expects a wxString